VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -220,12 +230,22 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
         if (!participant) {
             return;
         }
+        if (participant instanceof LinkedParticipantPanelModel && participant.hearingRole === HearingRole.INTERPRETER) {
```

Comment:
As far as I can tell this if statement will never be satisfied as this is in LinkedParticipantPanelModel. Note lines 14 and 19.

![image](https://user-images.githubusercontent.com/21690021/148206607-4ee603a0-d695-4ab1-a7c8-1a3426eed896.png)

If this is hit there may be somewhere else this is done differently that I don't know about where we force it instead of using the static methods in that class.

Reply:
We are not checking for an interpreter anymore